src/store/rule/types.ts

```diff
@@ -1,11 +1,13 @@
 import {Group} from "../group/types";
 
 export interface Rule {
-  ID?: string
-  Name: string
-  Source: Group[] | string[] | null
-  Destination: Group[] | string[] | null
-  Flow: string
+    ID?: string
+    Name: string
+    Description: string
+    Source: Group[] | string[] | null
+    Destination: Group[] | string[] | null
+    Flow: string
+    Disabled: boolean
```

Comment:
Codacy has a fix for the issue: [Replace `····Disabled:·boolean` with `··Disabled:·boolean;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  Disabled: boolean;
```